Rakefile

```
@@ -262,6 +263,399 @@ task :xcode => [:dependencies] do
   sh "open #{XCODE_WORKSPACE}"
 end
 
+desc "Install and configure WordPress iOS and it's dependencies - External Contributors"
+namespace :init do
+task :oss => %w[
+  install:xcode:check
+  dependencies
+  install:tools:check_oss
+  install:lint:check
+  credentials:setup
+]
+
+desc "Install and configure WordPress iOS and it's dependencies - a8c Developers"
```

```suggestion
desc "Install and configure WordPress iOS and it's dependencies - Automattic Developers"
```

Before joining, I wasn't familiar with what A8C meant.

---

Fixed